ScaleBlobs Operator

bilgeacun opened the pull request:
Summary:
Implementation of aggregated Scale operator.
The operator takes a list of tensors as an input and scales all of them them with the argument float value.
The tensor sizes can be different, therefore bookkeeping of the sizes and pointers to the tensors are
necessary for the GPU version of the kernel.

Differential Revision: D14984233

xiaomengy reviewed (approved):
It seems the failed amd test is not related.

bilgeacun committed:
ScaleBlobs Operator (#19660)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/19660

Implementation of aggregated Scale operator.
The operator takes a list of tensors as an input and scales all of them them with the argument float value.
The tensor sizes can be different, therefore bookkeeping of the sizes and pointers to the tensors are
necessary for the GPU version of the kernel.

Reviewed By: BIT-silence

Differential Revision: D14984233

fbshipit-source-id: 526dedbfecd316a652a8d80411f236f41afb203a
